src/lwc/gsChecklistIcon/gsChecklistIcon.html

```diff
@@ -0,0 +1,20 @@
+<template >
+    <div class="slds-media__figure">
+        <div class="slds-progress-ring slds-progress-ring_large">
```

Comment:
I suspect you can simplify this significantly by using [<lightning-progress-ring>](https://developer.salesforce.com/docs/component-library/bundle/lightning-progress-ring/example).

Reply:
I don't understand this comment

Reply:
Was there more to this comment, @force2b ?

Reply:
Yes. Sorry. It should have said:
I suspect you can simplify this significantly by using [\<lightning-progress-ring\>](https://developer.salesforce.com/docs/component-library/bundle/lightning-progress-ring/example).